tests/queries/0_stateless/03275_auto_cluster_functions_with_parallel_replicas.sql

```diff
@@ -0,0 +1,16 @@
+-- Tags: no-fasttest
+-- Tag no-fasttest: Depends on AWS
+
+SET enable_parallel_replicas=1, max_parallel_replicas=3, parallel_replicas_for_non_replicated_merge_tree=1, cluster_for_parallel_replicas='test_cluster_two_shard_three_replicas_localhost';
```

Comment:
Right now `max_parallel_replicas` is not used in IStorageCluster - so it will not limit the number of used shards.